sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,995 @@
+left for future work, but is summarized in a separate section after Normal mode.
+
+## Deposit
+
+To deposit bitcoin into the system a transaction will need to be generated
+that sends the UTXO to the generated address with two script spending paths.
+One with a special payload in an `OP_DROP` unlock script will need to be sent to
+the signer's pubkey, and another with a timelocked refund path. Then, a function
+with within the `.sbtc` smart contract will need to be called that does the following:
+
+1. Verifies that the Bitcoin transaction is included in the bitcoin block of a specified height
+2. Verifies the inclusion of the 2 script paths tap leaves in the output (the root is generated from the 2 tap leaves)
+3. Records the existence of the Transaction ID, the amount sent and the time at which it was committed in the `.sbtc` data space and marked as _pending_ (the height at which this Bitcoin transaction was included, not at which it was verified in the Clarity contract)
+
+Stackers, via their signer daemons, already subscribe to the Stacks blockchain
+for events.  The act of writing new sBTC deposit state to the `.sbtc` contract
+generates such an event -- an event that alerts the signer daemons to begin a
+signing round to act on the deposit.
+
+In the happy path, at least 70% of Stackers agree that the deposit transaction is
+valid, and they generate a contract-call to the `.sbtc` contract to
+_authorize_ the deposit by minting the equivalent number of sBTC tokens to
+an account address of the user's choice.  Crucially, the transaction is signed
+collectively by Stackers via a WSTS signing round -- it is authenticated via a
+Schnorr proof and the Stacker's aggregate public key.  This way, _only_ the
+Stackers can authorize the deposit, and only via a 70%+ majority vote.
+
+Once stackers generate the authorization transaction, they make it available to
+miners via their StackerDB instance, and require the current miner to include it
+in the next block they produce.  If the miner does not do so, then the Stackers
+will not sign the block; this all but compels the miner to include the
+transaction.
+
+The user must wait until their deposit transaction confirms on Bitcoin before the
+sBTC tokens materialize.  This is required because Stacks miners only process
+pending Stacks-on-Bitcoin transactions for Bitcoin blocks up to but _excluding_
+the Bitcoin block in which their `block-commit` landed.
+
+The act of fulfilling a deposit with an affirmative vote is effectively a promise
+made by Stackers to users to ensure that the sBTC minted wll remain fully backed
+by the BTC the user submitted.  This BTC may only be spent to fulfill withdrawals,
+or to process a DKG wallet hand-off.  Either way, the user's BTC will be spent by
+Stackers before they can reclaim it via the spending fulfillment script.
+
+***TODO: https://github.com/stacks-network/sbtc/issues/349: Create Deposit Procedure Diagram***
```

Comment:
@FriendsFerdinand can you address this ticket?

https://github.com/stacks-network/sbtc/issues/349